notebooks/features/classification/Classification - Before and After SynapseML.ipynb

```diff
@@ -1,14 +1,45 @@
 {
  "cells": [
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Classification - Before and After SynapseML\n",
-    "\n",
-    "### 1. Introduction\n",
+    "# Classification - Before and After SynapseML"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "synapse_internal_only"
+    ]
+   },
+   "source": [
+    "## Prerequisites\n",
     "\n",
-    "<p><img src=\"https://images-na.ssl-images-amazon.com/images/G/01/img16/books/bookstore/landing-page/1000638_books_landing-page_bookstore-photo-01.jpg\" style=\"width: 500px;\" title=\"Image from https://images-na.ssl-images-amazon.com/images/G/01/img16/books/bookstore/landing-page/1000638_books_landing-page_bookstore-photo-01.jpg\" /><br /></p>\n",
+    "* Attach your notebook to a lakehouse. On the left side, select **Add** to add an existing lakehouse or create a lakehouse."
```

Comment:
Lets factor this out into a common installation and setup doc so its not in every NB

Reply:
move it to manifest file to make it platform independent.